security/response-process.md

```diff
@@ -0,0 +1,245 @@
+| Feature request     | Let the reporter know this is the intended behavior. If they think this behavior could be improved, they can file a feature request. Close the security issue.                                                                                                                                                                                                         |
+| Vulnerability       | Let the reporter know that you have confirmed this unwanted behavior creates a security issue. Proceed with the process.                                                                                                                                                                                                                                               |
+| Out of scope        | Let the reporter know that the issue lies in upstream code packaged by OQS. Assist them in refiling the report upstream. See the following section for further actions.                                                                                                                                                                                                  |
+
+#### "Out of scope" assessments
+
+OQS packages cryptographic algorithm implementations from a variety of upstream sources.
+These vary widely in level of support and software development expertise.
+OQS assumes no obligation to fix vulnerabilities in upstream code, but it may choose to do so on a best-effort basis.
+
+For all "out of scope" assessments, the vulnerability report should be passed on to the relevant upstream(s).
+Depending on the nature of the issue, additional action may be required.
+
+| "Out of scope" assessment | Description | Additional action |
+|------------------|----------|-|
+| Unable to assess | Assessing the issue requires domain knowledge about the upstream beyond that of the OQS VMT. | Monitor the upstream for resolution. |
+| Vulnerability: can patch  | The issue is indeed a vulnerability and can be simply and easily patched within OQS. | Decide whether or not to develop a patch. Coordinate disclosure with the upstream. |
+| Vulnerability: can't patch | The issue is indeed a vulnerability but cannot be simply and easily patched within OQS. | Coordinate disclosure with the upstream. Monitor the upstream for resolution. Depending on severity, consider dropping the upstream. |
+
+Decisions on whether or not to proceed with a possible patch should be made taking into account specifics of the upstream source.
+These include responsiveness to vulnerability reports and ability of the upstream maintainers to develop a patch.
+It is also important to consider the impact on end-users.
+A longer response time can be tolerated for an experimental algorithm than for a standardized or standard-track algorithm.
```

Comment:
Where do we spell out which algorithms fall into which category?

Reply:
I've created https://github.com/open-quantum-safe/liboqs/issues/2045 to track this.